dns/dnssec.py

```diff
@@ -493,6 +493,23 @@ def _need_pycrypto(*args, **kwargs):
     validate_rrsig = _need_pycrypto
     _have_pycrypto = False
     _have_ecdsa = False
+
+    import hashlib
```

Comment:
I also see no reason to keep using the `Crypto` or `Cryptodome` package for hashing functions either. For sake of simplicity and consistency, let's use hashlib everywhere.